index.js

```diff
@@ -4,9 +4,10 @@ var xtend = require('xtend');
 var ShelfPack = require('shelf-pack');
 var queue = require('queue-async');
 var emptyPNG = new mapnik.Image(1, 1).encodeSync('png');
-var sortBy = require('sort-by');
 
-var heightAscThanNameComparator = sortBy('-height', 'id');
+function heightAscThanNameComparator(a, b) {
+    return (b.height - a.height) || (a.id < b.id ? -1 : 1);
```

Comment:
Just a thought: this comparator isn't transitive as implemented, it shouldn't happen but if a and b were the same object then this would likely result in undefined behavior. ie: `a > b` is true but `b > a`  is also true. 

Would be safer to have: `(a.id === b.id) ? 0 : (a.id < b.id ? -1 : 1);`